src/main/java/EmployeesInfo.java

```diff
@@ -0,0 +1,44 @@
+public class EmployeesInfo {
+
+    public static double averageSalary(Employee[] employees) {
+        double sumSalary = 0;
+        for (int i = 0; i < employees.length; i++) {
+            sumSalary += employees[i].getSalary();
+        }
+        return sumSalary / employees.length;
+    }
+
+    public static int minSalary(Employee[] employees) {
+        int minSalary = employees[0].getSalary();
+        for (int i = 0; i < employees.length; i++) {
+            if (employees[i].getSalary() < minSalary) {
+                minSalary = employees[i].getSalary();
+            }
+        }
+        return minSalary;
+    }
+
+    public static int maxSalary(Employee[] employees) {
+        int maxSalary = employees[0].getSalary();
+        for (int i = 0; i < employees.length; i++) {
+            if (employees[i].getSalary() > maxSalary) {
+                maxSalary = employees[i].getSalary();
+            }
+        }
```

Comment:
Jeśli celem pętli jest przejście po wszystkich elementach tablicy, to lepiej zastosować pętlę foreach https://youtu.be/Tx1A6Wo6UX0